egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -322,6 +326,7 @@ if ! "${skip_data_prep}"; then
 
         log "Stage 2: Format wav.scp: data/ -> ${data_feats}/"
         for dset in "${train_set}" "${valid_set}" ${test_sets}; do
+            continue
```

Comment:
Do not forget to remove this after final submit.

Reply:
Oh, many thanks for noting it!